[2024-05-12T20:20:52.126+0500] {processor.py:161} INFO - Started process (PID=9386) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:20:52.126+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:20:52.127+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:20:52.127+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:20:52.130+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:20:52.129+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:20:52.130+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:20:52.130+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:20:52.130+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:20:52.137+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-05-12T20:21:46.681+0500] {processor.py:161} INFO - Started process (PID=9634) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:21:46.682+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:21:46.684+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:21:46.683+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:21:46.686+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:21:46.686+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:21:46.686+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:21:46.686+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:21:46.687+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:21:46.694+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-05-12T20:22:35.799+0500] {processor.py:161} INFO - Started process (PID=9836) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:22:35.800+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:22:35.802+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:22:35.802+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:22:35.805+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:22:35.805+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:22:35.806+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:22:35.806+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:22:35.806+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:22:35.817+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-05-12T20:23:32.479+0500] {processor.py:161} INFO - Started process (PID=10311) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:23:32.481+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:23:32.483+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:23:32.482+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:23:32.486+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:23:32.485+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:23:32.486+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:23:32.486+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:23:32.487+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:23:32.503+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2024-05-12T20:24:18.453+0500] {processor.py:161} INFO - Started process (PID=10568) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:24:18.455+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:24:18.456+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:24:18.456+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:24:18.458+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:24:18.458+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:24:18.459+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:24:18.459+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:24:18.459+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:24:18.466+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-05-12T20:25:18.211+0500] {processor.py:161} INFO - Started process (PID=10850) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:25:18.213+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:25:18.214+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:25:18.214+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:25:18.217+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:25:18.217+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:25:18.217+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:25:18.217+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:25:18.218+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:25:18.226+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-05-12T20:26:20.765+0500] {processor.py:161} INFO - Started process (PID=11168) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:26:20.766+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:26:20.767+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:26:20.767+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:26:20.770+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:26:20.769+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:26:20.770+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:26:20.770+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:26:20.770+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:26:20.777+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-05-12T20:27:56.980+0500] {processor.py:161} INFO - Started process (PID=12002) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:27:56.996+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:27:56.998+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:27:56.997+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:27:57.000+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:27:57.000+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:27:57.001+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:27:57.001+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:27:57.001+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:27:57.008+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-05-12T20:28:48.777+0500] {processor.py:161} INFO - Started process (PID=12247) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:28:48.778+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:28:48.780+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:28:48.780+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:28:48.782+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:28:48.782+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:28:48.782+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:28:48.782+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:28:48.783+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:28:48.790+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-05-12T20:29:42.370+0500] {processor.py:161} INFO - Started process (PID=12560) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:29:42.372+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:29:42.373+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:29:42.373+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:29:42.376+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:29:42.376+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:29:42.376+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:29:42.376+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:29:42.376+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:29:42.385+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-05-12T20:31:02.738+0500] {processor.py:161} INFO - Started process (PID=13158) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:31:02.766+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:31:02.768+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:31:02.767+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:31:02.771+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:31:02.770+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:31:02.771+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:31:02.771+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:31:02.772+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:31:02.780+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-05-12T20:31:54.640+0500] {processor.py:161} INFO - Started process (PID=13396) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:31:54.642+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:31:54.644+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:31:54.644+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:31:54.649+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:31:54.648+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:31:54.649+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:31:54.649+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:31:54.650+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:31:54.662+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2024-05-12T20:34:36.963+0500] {processor.py:161} INFO - Started process (PID=14572) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:34:36.965+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:34:36.966+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:34:36.965+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:34:36.969+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:34:36.968+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:34:36.969+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:34:36.969+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:34:36.969+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:34:36.978+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-05-12T20:35:29.735+0500] {processor.py:161} INFO - Started process (PID=14855) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:35:29.763+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:35:29.764+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:35:29.764+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:35:29.768+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:35:29.767+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:35:29.768+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:35:29.768+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:35:29.769+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:35:29.778+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2024-05-12T20:36:23.757+0500] {processor.py:161} INFO - Started process (PID=15125) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:36:23.784+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:36:23.784+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:36:23.784+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:36:23.787+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:36:23.787+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:36:23.788+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:36:23.788+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:36:23.788+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:36:23.797+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-05-12T20:37:11.040+0500] {processor.py:161} INFO - Started process (PID=15367) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:37:11.042+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:37:11.043+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:37:11.042+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:37:11.045+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:37:11.045+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:37:11.045+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:37:11.045+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:37:11.046+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:37:11.054+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-05-12T20:39:10.448+0500] {processor.py:161} INFO - Started process (PID=16386) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:39:10.476+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:39:10.477+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:39:10.477+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:39:10.480+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:39:10.480+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:39:10.480+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:39:10.480+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:39:10.481+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:39:10.494+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-05-12T20:40:03.444+0500] {processor.py:161} INFO - Started process (PID=16711) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:40:03.445+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:40:03.446+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:40:03.446+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:40:03.448+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:40:03.448+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:40:03.449+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:40:03.449+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:40:03.449+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:40:03.459+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-05-12T20:40:55.144+0500] {processor.py:161} INFO - Started process (PID=17079) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:40:55.189+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:40:55.190+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:40:55.189+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:40:55.192+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:40:55.192+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:40:55.193+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:40:55.193+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:40:55.193+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:40:55.202+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-05-12T20:41:39.401+0500] {processor.py:161} INFO - Started process (PID=17347) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:41:39.420+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:41:39.421+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:41:39.421+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:41:39.426+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:41:39.425+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:41:39.426+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:41:39.426+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:41:39.426+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:41:39.435+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2024-05-12T20:42:25.696+0500] {processor.py:161} INFO - Started process (PID=17643) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:42:25.712+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:42:25.713+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:42:25.713+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:42:25.715+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:42:25.715+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:42:25.715+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:42:25.715+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:42:25.716+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:42:25.724+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-05-12T20:43:30.942+0500] {processor.py:161} INFO - Started process (PID=18099) to work on /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:43:30.972+0500] {processor.py:830} INFO - Processing file /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-12T20:43:30.973+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:43:30.973+0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:43:30.975+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:43:30.975+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-12T20:43:30.976+0500] {logging_mixin.py:188} INFO - [2024-05-12T20:43:30.975+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-12T20:43:30.976+0500] {processor.py:842} WARNING - No viable dags retrieved from /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-12T20:43:30.989+0500] {processor.py:183} INFO - Processing /home/awahab02/mlops_assignment_2/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
